timber/src/main/java/timber/log/Timber.java

```diff
@@ -116,6 +117,30 @@ public static void plant(Tree tree) {
     }
   }
 
+  /** Add a new logging tree. */
+  public static void plant(Tree... trees) {
+    if (trees == null) {
+      throw new NullPointerException("trees == null");
+    }
+    if (trees.length == 0) {
+      return;
+    }
+    //noinspection ForLoopReplaceableByForEach
+    for (int i = 0; i < trees.length; i++) {
```

Comment:
Ah I see — in that case, why doesn't this use a for-each https://github.com/JakeWharton/timber/blob/master/timber/src/main/java/timber/log/Timber.java#L161?